src/snabbdom.ts

```diff
@@ -60,10 +60,38 @@ export function init(modules: Array<Partial<Module>>, domApi?: DOMAPI) {
     }
   }
 
-  function emptyNodeAt(elm: Element) {
-    const id = elm.id ? '#' + elm.id : '';
-    const c = elm.className ? '.' + elm.className.split(' ').join('.') : '';
-    return vnode(api.tagName(elm).toLowerCase() + id + c, {}, [], undefined, elm);
+  function toVNode(elm: Node): VNode {
+    let text: string;
+    if (api.isText(elm)) {
```

Comment:
Nitpicking but I suspect `toVNode` will be called with an Element rather than a text or comment, most of the time. Wouldn't it make more sense to start with `if (api.isElement(elm))`?